### ISSUE TITLE
[feature] - provide package.json overrides

### ISSUE THREAD

**Issue by ecwyne** [onmodulus/demeteorizer#114]
When an application has dependency conflicts, it would be nice to be able to automatically define which version to use. I propose creating a `demeteorizer.json` file that contains overrides. Once the contents of `demeteorizer.json` are read and parsed, all that would have to be done is extending the `packageJSON` object [here](https://github.com/onmodulus/demeteorizer/blob/master/lib/demeteorizer.js#L404) before `package.json` is written. 

pseudocode
```javascript
  packageJSON = deepExtend(demeterizerJSON, packageJSON);
```

## demeteorizer.json
```json
{
  "dependencies": {
    "underscore": "1.8.2"
  }
}
```

**Comment by fiveisprime** [comment]
This doesn't really work in the updated version of demeteorizer. Here's why: the
actual npm modules that are included for each Meteor package are nested way down
deep in the bundle while the package.json that's generated for the bundle is up
in the root.

The tree will end up like this

    .
    ├── bundle
    │   ├── README
    │   ├── main.js
    │   ├── programs
    │   │   ├── server
    │   │   │   ├── app
    │   │   │   │   └── leaderboard.js
    │   │   │   ├── assets
    │   │   │   │   └── packages
    │   │   │   │       └── boilerplate-generator
    │   │   │   │           ├── boilerplate_web.browser.html
    │   │   │   │           └── boilerplate_web.cordova.html
    │   │   │   ├── boot-utils.js
    │   │   │   ├── boot.js
    │   │   │   ├── config.json
    │   │   │   ├── mini-files.js
    │   │   │   ├── npm <<<<<<<<<<<<<<<<<<<<<< directory where meteor stores npm modules
    │   │   │   │   ├── ddp
    │   │   │   ├── npm-shrinkwrap.json
    │   │   │   ├── package.json <<<<<<<<<<<<< generated package.json
    │   │   │   ├── packages
    │   │   │   │   ├── autopublish.js
    ...

With that tree structure, running npm install on the generated package creates a
`node_modules` directory in the server directory which is not where `main.js`
will be looking.

In other words, the crazy Meteor directory structure makes this particular
feature useless as the dependencies are no longer managed through npm even with
demeteorizer.
